1.12/Dockerfile.fedora

```diff
@@ -0,0 +1,40 @@
+FROM registry.fedoraproject.org/f30/s2i-base:latest
```

Comment:
The latest s2i-base contains also fix with `/usr/libexec/platform-python` https://bodhi.fedoraproject.org/updates/FEDORA-CONTAINER-2020-75cda7f6b8 although it is not related to golang container.

Reply:
F31 has golang 1.13 :/